Add a student's major and class level to the User table

sreynit02 opened the pull request:
Fixes issue #854 
Did: 
- added "major" and "classLevel" to User table
- configure import_users.py so it can pull these two information from Tracy to our database

Test: 
- reset database
- update the Tracy password in config/default.yml (ask Brian or Sreynit or Fleur for password)
- In database run 'python import_users.py'
- Check the User table. "major and classLevel" for students should be populated with their major and class level.

sreynit02 committed:
pull major and classLevel for student in User table

sreynit02 committed:
import major and classlevel

AndersonStettner reviewed (changes requested):
add pyodbc to requirements.txt and the students who are in base data are not being updated correctly. 
EX: I changed Sreynit's name to just Sr, re-ran import_users and it fixed the name but still did not add a class level or major, it is like that for all students in base data.

gahimbaref committed:
update existing user in db

sreynit02 commented:
> add pyodbc to requirements.txt and the students who are in base data are not being updated correctly. EX: I changed Sreynit's name to just Sr, re-ran import_users and it fixed the name but still did not add a class level or major, it is like that for all students in base data.

fixed!

AndersonStettner reviewed (changes requested):
Nevermind, I went ahead and made the change since it was so small, everything looks good!

AndersonStettner committed:
resolved only coment
